### PR TITLE
feat(sdk): remove fields marked with internal comment

### PR DESCRIPTION
**Pull request by littlemooon**

### CODE REVIEW

File: packages/codegen-doc/src/field.ts [hunk 1]
```diff
@@ -13,15 +13,17 @@ export function isScalarField(context: PluginContext, field: FieldDefinitionNode
  * Determine whether to the field is valid to be output
  *
  * Use config skipFields to skip fields with specific names
+ * Use config skipComments to skip fields with comments containing specific strings
  * Use config skipDirectives to skip fields with specific directives
  */
 export function isValidField(context: PluginContext, field?: FieldDefinitionNode): field is FieldDefinitionNode {
   if (field && field.name.value !== "edges") {
     const skipFieldName = context.config.skipFields?.includes(field?.name?.value ?? field?.name ?? "");
+    const skipComment = context.config.skipComments?.some(comment => field.description?.value.includes(comment));
```
Comment (littlemooon) [inline packages/codegen-doc/src/field.ts:22]: skip fields containing a skipped comment string

File: packages/codegen-doc/src/types.ts [hunk 2]
```diff
@@ -47,6 +47,24 @@ export interface PluginConfig {
...
+   *     plugins:
+   *       - "@linear/codegen-doc"
+   *     config:
+   *       skipComments:
+   *         - "[Internal]"
+   *         - "-- Deprecated --"
+   *
+   * ```
+   */
+  skipComments?: string[];
```
Comment (littlemooon) [inline packages/codegen-doc/src/types.ts:67]: `skipComments` config definition

File: packages/sdk/codegen.doc.yml [hunk 3]
```diff
@@ -4,6 +4,8 @@ hooks:
   afterOneFileWrite:
     - prettier --write
 config:
+  skipComments:
+    - "[Internal]"
```
Comment (littlemooon) [inline packages/sdk/codegen.doc.yml:8]: `[Internal]` configuration

**Review by eldh — APPROVED** [review]
Sweet

**Pull request merged**